en/ros2/px4_sdk.md

```diff
@@ -0,0 +1,336 @@
+# PX4 SDK
+
+The [PX4 SDK](https://github.com/PX4/px4_sdk) is a C++ library to simplify controlling PX4 from ROS 2.
+It allows to write external modes that are dynamically registered with PX4 and behave the same way as internal ones.
+A mode can send different types of setpoints, ranging from high-level navigation tasks all the way down to direct actuator controls.
+
+## Overview
```

Comment:
You could consider omitting this heading and moving all others up a level. If you do this you can't add too much here, but you could turn "### Comparison to Offboard Control" into a few sentences and put that here too. So "These modes are first class citizens blah blah. This approach offers many benefits over conventional offboard control because ...".

Just a thought.